### ISSUE TITLE
doc: Quick Configuration page is wrong/outdated

### ISSUE THREAD

**Issue by g-a-c** [git-town/git-town#2034]
The Quick Configuration page refers to installing aliases as `git town install aliases true` and removing aliases with `git town install aliases false`

However these commands result in:

```
$ git town install aliases
Error: unknown command "install" for "git-town"
Run 'git-town --help' for usage.

Error: unknown command "install" for "git-town"
```

**Comment by g-a-c** [comment]
- https://www.git-town.com/quick-configuration.html contains `git town install aliases true/false`
- https://www.git-town.com/commands/install-aliases.html contains `git town aliases add/remove`

Neither of these seem to be right for `v7.9.0` that I have installes, it requires `git town alias true/false`

I will submit a PR to update these two documentation pages to make the behaviour I have observed with `v7.9.0`

**Comment by g-a-c** [comment]
OK, this is starting to look like a bigger issue...

My version of Git Town reports as

```
$ git town version
Git Town v7.9.0 (2022/12/28)
```

Which makes sense, however there appears to have been considerable doc changes committed to the `main` branch since December 28th. It _looks_ to me (an outsider, apologies if I'm misunderstanding how your project works) that these doc changes should probably have been released with a new version but that they may have been tagged with an incorrect tag.

Git tags exist for both `v7.9.0` (2022-12-28) **and** `7.9.0` (2023-01-22). The new docs for (e.g.) `git install alias` are present in main, but my Homebrew installation of `v7.9.0` does not have it. These new docs were added recently (`git diff v7.9.0..7.9.0` shows new references to `git town install`).

Is it possible that someone mistagged this release as `7.9.0` on `main` when it **should** have been tagged `v7.10.0` instead?

**Comment by g-a-c** [comment]
Apologies for the comment spam, but I think I see the issue.

I installed from [Homebrew](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/git-town.rb). This pulls the code from `v7.9.0.tar.gz`. This release does not show as a GitHub release on your Releases page in this repo - however the tag **does** exist, so this tar file can be downloaded and built. This would be running the code from 2022-12-28, which does not contain the documentation fixes currently on `main` and therefore the current documentation on the Git Town website isn't accurate

I will submit a PR to the Homebrew project and link them to this issue, and close my issue/PR in this project as the documentation is correct, I just need to get the right code onto my Mac to reflect those docs.

**Comment by kevgo** [comment]
Thanks for reporting this. We are currently preparing a larger refactor of the Git Town commands that will be released as Git Town 8.0 since it contains some breaking changes. An example is the one you stumbled upon. The docs were already updated to the new behavior but the code hasn't been released yet. I hope to get to it in the foreseeable future.

**Comment by kevgo** [comment]
There were indeed two tags for the 7.9.0 release: `7.9.0` and `v7.9.0`. I have cleaned up the wrong tag. Thanks for reporting this! :green_heart:
